pkg/remote/aws/sqs_queue_policy_enumerator.go

```diff
@@ -0,0 +1,48 @@
+package aws
+
+import (
+	"github.com/cloudskiff/driftctl/pkg/remote/aws/repository"
+	remoteerror "github.com/cloudskiff/driftctl/pkg/remote/error"
+	"github.com/cloudskiff/driftctl/pkg/resource"
+	"github.com/cloudskiff/driftctl/pkg/resource/aws"
+
+	awssdk "github.com/aws/aws-sdk-go/aws"
+)
+
+type SqsQueuePolicyEnumerator struct {
```

Comment:
Can you change everywhere to `SQS` and not `Sqs`. Thanks !

Reply:
i have changed Sqs to SQS everywhere in the project on another PR. Let me know if it's too much and you prefer to change only on this PR, but for example on #750 it's still Sqs so i think it's not clean to change only on #749.